### PR TITLE
Add tooltip to showed full balance in token detail screen

### PR DESCRIPTION
**Pull request by youngkidwarrior**

### CODE REVIEW

**Comment by youngkidwarrior** [comment]
> [!WARNING]
> <b>This pull request is not mergeable via GitHub because a downstack PR is open. Once all requirements are satisfied, merge this PR as a stack <a href="https://app.graphite.dev/github/pr/0xsend/sendapp/357?utm_source=stack-comment-downstack-mergeability-warning" >on Graphite</a>.</b>
> <a href="https://graphite.dev/docs/merge-pull-requests">Learn more</a>

* **#357** <a href="https://app.graphite.dev/github/pr/0xsend/sendapp/357?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* **#356** <a href="https://app.graphite.dev/github/pr/0xsend/sendapp/356?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#355** <a href="https://app.graphite.dev/github/pr/0xsend/sendapp/355?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#354** <a href="https://app.graphite.dev/github/pr/0xsend/sendapp/354?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#353** <a href="https://app.graphite.dev/github/pr/0xsend/sendapp/353?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* `main`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @youngkidwarrior and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

**Review by 0xBigBoss — APPROVED** [review]
lgtm let's just drop the logs

File: packages/app/features/home/TokenDetails.tsx [hunk 1]
```diff
@@ -74,13 +75,16 @@ export const TokenDetails = ({ coin }: { coin: coins[number] }) => {
 
 export const TokenDetailsMarketData = ({ coin }: { coin: coins[number] }) => {
   const { data: tokenMarketData, status } = useTokenMarketData(coin.coingeckoTokenId)
+  console.log('tokenMarketData: ', tokenMarketData)
```
[the remark this answers is not in the document]
Comment (youngkidwarrior) [reply packages/app/features/home/TokenDetails.tsx:78]: dammit

**Comment by youngkidwarrior** [comment]
### Merge activity

* **Apr 26, 6:54 PM EDT**: @youngkidwarrior started a stack merge that includes this pull request via [Graphite](https://app.graphite.dev/github/pr/0xsend/sendapp/357).

**Pull request merged**